source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,361 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disaster Recovery Process
+last_reviewed_on: 2024-01-24
+review_in: 6 months
+---
+
+
+# <%= current_page.data.title %>
+
+
+This page is designed to help give direction in the event of a DR scenario.
+
+
+It covers permanent loss of all infrastructure, temporary loss of all infrastructure, and loss of a single account.
+
+
+## Diagnose
+
+
+To find out the severity of the situation, check the AWS status page [here](https://health.aws.amazon.com/health/status ).
+
+
+Regardless of which type of disaster recovery situation there is, you should contact AWS **immediately**, understanding if this is an outage for 1 hour or 1 month will dictate the next steps to take.
+
+
+- If the issue is across all accounts, login to the Modernisation Platform account, and raise the highest level severity ticket, whilst also enganging with our AWS representative in the `#ext-aws` channel.
+
+
+- If this issue is in one account, login to that account, and raise the highest level severity ticket if production, and also enganging with our AWS representative in this channel **fill in**
+
+
+- If this occurs is in hours, create a huddle / meet. Outside of hours, call the second member of staff on call and post to the channel. With the PagerDuty alerts, incident channels will be created, too many to follow, so make a lead channel to coordinate.
+
+
+## Internal Escelation
+
+
+**Simon can you help fill in this**
+
+
+Inform the following channels of the situation and put forward a contactable member of staff from our team.
+
+
+`#ask-modernisation-platform`
+
+
+`#modernisation-platform`
+
+
+`#modernisation-platform-alerts`
+
+
+
+## Disaster Recovery Types
+
+
+**Let’s work the problem, people. Let’s not make things worse by guessing.**
+
+
+- London has been lost but **no services need to migrate**. [Click here]().
+- London has been lost, and **all services need to migrate**. [Click here]().
+- Loss of a **single account**. [Click here]().
+
+
+## London has been lost but returning soon.
+
+
+### Getting a decision
+
+
+Once a ticket has been raised with AWS, and we have engaged with them and have total confirmation that the infrastructure will be returning within X number of minutes or hours, we will need to pass this information on for an informed decision to be made above.
+
+
+Only when we have this, can we update users on what is next.
```

Comment:
I agree, but should we have a commitment here to update every x minutes/hours - even if we are still waiting?

Reply:
Ive added a line to say this needs to be filled in

Reply:
I think the confusion is that it reads a bit that we wouldn't engage with users until we know how long it will take for the issue to resolve.

Reply:
Just to jump onto Ewa's point. Would we inform the users first (via a slack channel or directly?) that we are aware that there is an incident and we are looking into it with AWS. Just so we are notifying them before we ask AWS.